exporter/signalfxexporter/translation/translator.go

```diff
@@ -327,6 +336,47 @@ func (mp *MetricTranslator) TranslateDataPoints(logger *zap.Logger, sfxDataPoint
 					convertMetricValue(logger, dp, newType)
 				}
 			}
+		case ActionDivideMetrics:
+			var numerator, denominator *sfxpb.DataPoint
+			for _, dp := range processedDataPoints {
+				// there should only be one entry in tr.Mapping
+				for nName, dName := range tr.Mapping {
+					if dp.Metric == nName {
+						numerator = dp
+					} else if dp.Metric == dName {
+						denominator = dp
+					}
+				}
+			}
+
+			if numerator == nil {
+				logger.Warn("divideMetrics: numerator == nil")
+				continue
+			}
+			if numerator.Value.IntValue == nil {
```

Comment:
Might be nice to add support for (int or float) divided by (int or float) rather than the current (int) divided by (int). Can be done in a separate PR.

Reply:
We can add more functionality when we need it. if we need int only atm, let's stick with them